src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1221,6 +1644,33 @@ protected override void Dispose(DisposeTypes type)
             rotationGestureDetector?.Dispose();
             rotationGestureDetector = null;
 
+#if NUI_PROPERTY_CHANGE_1
```

Comment:
Is this NUI_PROPERTY_CHANGE_1 or NUI_PROPERTY_CHANGE_3?
Because internalCurrentParentOrigin is defined with NUI_PROPERTY_CHANGE_3.

Reply:
thank you for review.
you are right, (3 is right)
it is fixed.